packages/core/src/validate.js

```diff
@@ -128,10 +131,11 @@ function unwrapErrorHandler(errorHandler) {
   return Object.keys(errorHandler).reduce((acc, key) => {
     if (key === "addError") {
       return acc;
-    } else if (key === "__errors") {
+    } else if (isPlainObject(errorHandler[key])) {
```

Comment:
Should this be `key === "__errors" && isPlainObject(errorHandler[key])`? Why remove the `key === "__errors"` part?